Automattic/jetpack
# Social: Improve active plugin checks when activating a Social license.

## Proposed changes:

* Called the wp/v2/plugins core end-poin to get the list of active plugins and removed our implementation of isPluginActive from the util. 

### Other information:


## Jetpack product discussion


pdWQjU-Hw-p2#comment-746
## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






- Boot this branch on JN
- Buy Jetpack Social Basic, by going to https://cloud.jetpack.com/pricing/
- Go to your site's wp-admin/admin.php?page=my-jetpack#/add-license page to activate the license you just bought.
- You should see the configure my site button. If the button says 'view my plan' something is wrong.

<img width="888" alt="Screenshot 2024-02-01 at 6 30 45 PM" src="https://github.com/Automattic/jetpack/assets/6594561/e9417c54-2bc7-4289-a19f-dd3a4cb0a94b">

- Do a siteless checkout of another Social Advanced plan. 
- Apply the advanced plan license on the same site, You should see the configure my site button.
